.travis.yml

```diff
@@ -77,7 +80,22 @@ matrix:
       services:
         - docker
       script:
-        - "./build/travis/job2_AppImage/build.sh --32bit --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
+        - "./build/travis/job2_AppImage/build.sh --i686 --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
+
+    # 4th parallel build job - portable Linux AppImage armhf build on Debian crosscompiler
+    - env: "JOB=AppImage_armhf"
+      addons:
+        apt:
+          packages:
+          - bsdtar
+          - curl
+          - zsync
+      services:
+        - docker
+      before_script:
+        - "sudo ./build/travis/job2_AppImage/set-binfmt-misc.sh"
+      script:
+        - "./build/travis/job2_AppImage/build.sh --armhf --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
```

Comment:
What do you think about if only trigger this command if specified in env var, like: [ containsString( $UPLOAD_ARCHS, armhf ) ] && ./build